README.md

```diff
@@ -117,45 +117,12 @@ with varying privilege levels.
 4. Copy mozillians-tests/variables.json to a location outside of mozillians-tests. Update the 'vouched', 'private', and 'unvouched' users in variables.json with those credentials
 
 #### Running tests locally
-Before each test run, clean up the repo:
-    find . \( -name 'results*' -or -name '*.pyc' \) -print0 | xargs -0 rm -Rf
 
-Note: If you are running tests on a Mac, run:
-    find . -name 'results*' -or -name '*.pyc' -print0 | xargs -0 rm -Rf
-
-To run tests locally it is as simple as calling <code>py.test</code> with
-several flags. To run testcases that do not modify or delete data:
-
-    py.test --driver=Firefox --base-url=http://mozillians.allizom.org --variables=/full/path/to/variables.json .
-
-__Output__
-
-Output of a test run should look something like this:
-
-    ============================= test session starts ==============================
-    collected 15 items
-
-    tests/test_about_page.py ..
-    tests/test_account.py ..
-    tests/test_invite.py ..
-    tests/test_profile.py ...x.x
-    tests/test_search.py ...
-    ==================== 13 passed, 2 xpassed in 172.03 seconds ====================
-
-__Note__
-"~" will not resolve to the home directory when used in the py.test command line.
-
-Some options for py.test are pre-specified by the file mozillians-tests/mozwebqa.cfg
-
-The mozwebqa plugin has advanced command line options for reporting and using
-browsers. See the documentation on [pytest mozwebqa github][pymozwebqa].
-[pymozwebqa]: https://github.com/mozilla/pytest-mozwebqa
-
-__Troubleshooting__
+Running Tests
```

Comment:
Remove the additional heading, and there's more we can remove above about installing dependencies and virtualenvs.